src/extension.web.ts

```diff
@@ -3,19 +3,14 @@
 
 'use strict';
 
-/* eslint-disable @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports */
-
-// This line should always be right on top.
-/* eslint-disable @typescript-eslint/no-explicit-any */
-if ((Reflect as any).metadata === undefined) {
-    require('reflect-metadata');
-}
+// reflect-metadata is needed by inversify, this must come before any inversify references
+import 'reflect-metadata';
```

Comment:
Same coment as earlier

Reply:
In hindsight I think we should have added comments around the conditional statement